tasks/release.py

```diff
@@ -713,6 +760,14 @@ def finish(
                 return Exit(code=1)
     print("datadog-agent-macos-build' tag is {}".format(_stringify_version(macos_build_version)))
 
+    if not windows_ddnpm_version:
+        # Get info on DDNPM
+        windows_ddnpm_driver, windows_ddnpm_version, windows_ddnpm_shasum = _get_windows_ddnpm_release_json_info(
```

Comment:
If `windows_ddnpm_version` is still `None`, what do you think about displaying an error message and exit like for `macos_build_version`? Is there a case where this field is optional?

Reply:
> If `windows_ddnpm_version` is still `None`, what do you think about displaying an error message and exit like for `macos_build_version`? Is there a case where this field is optional?

Hmm - I feel like `windows_ddnpm_version == None` is a very serious problem if we ever reach it. I believe that the `release.json` will already throw a missing key error if we can't pull out the windows_ddnpm_version from the right place.